x-pack/plugin/ml/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/LearningToRankRescorerIT.java

```diff
@@ -196,7 +197,6 @@ public void setupModelAndData() throws IOException {
         adminClient().performRequest(new Request("POST", INDEX_NAME + "/_refresh"));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/103072")
```

Comment:
What change fixes this test? Or should it have never been skipped?

Reply:
I'm not sure. I suspect @afoucret fixed it when he made all of the LeraningToRank changes.